### ISSUE TITLE
Is the pretrained model 20000.tar is trained on DTU dataset or the BlendedMVS dataset?

### ISSUE THREAD

**Issue by VILmchen** [jzhangbs/Vis-MVSNet#32]
Thanks for your nice work! 
Is the pre-trained model 20000.tar is trained on the DTU dataset or the BlendedMVS dataset? I directly use the pre-trained model 20000.tar and the sh/tnt.sh script and the fusion.py only get an F-score 55.97 on Tanks and Temples, is there any hyper-parameters that need to change?

**Comment by jzhangbs** [comment]
Hi, are you using the provided preprocessed TnT test set with provided sfm result instead of the one from colmap? Current fusion script can achieve 58.41 with the same setting.

**Comment by VILmchen** [comment]
Thanks for your reply. I use the preprocessed TnT test set which is downloaded from the repo of YoYo000/MVSNet. Besides, I want to confirm that is the pre-trained model 20000.tar is trained on the DTU dataset or the BlendedMVS dataset?

**Comment by jzhangbs** [comment]
it's trained on BlendedMVS

**Comment by VILmchen** [comment]
OK, thanks very much! Is there any methods to obtain the results shown on the leaderboard of TnT by the fusion script?

**Comment by jzhangbs** [comment]
We are working on a better re-implementation of the fusion code. We will release it soon.

**Comment by VILmchen** [comment]
OK, thanks!
